File: PHP/Program/Handphone.php
<?php
require_once "Sumisang.php";

class Handphone extends Sumisang {
    private $androidVer;
    private $uiVer;
    private $processor;

    // Constructor
    public function __construct($code, $name, $price, $androidVer, $uiVer, $processor, $photo) {
        parent::__construct($code, $name, $price, $photo);  // Menambahkan foto
        $this->androidVer = $androidVer;
        $this->uiVer = $uiVer;
        $this->processor = $processor;
    }

    // Setters and Getters
    public function setAndroidVer($androidVer) {
        $this->androidVer = $androidVer;
    }

    public function getAndroidVer() {
        return $this->androidVer;
    }

    public function setUiVer($uiVer) {
        $this->uiVer = $uiVer;
    }

    public function getUiVer() {
        return $this->uiVer;
    }

    public function setProcessor($processor) {
        $this->processor = $processor;
    }

    public function getProcessor() {
        return $this->processor;
    }
}
?>

File: PY/Program/Handphone.py
from Sumisang import Sumisang

class Handphone(Sumisang):
    def __init__(self, code="", name="", price=0, android_ver="", ui_ver="", processor=""):
        super().__init__(code, name, price) # memanggil konstruktor dari kelas induk
        # Private attributes
        self.__android_ver = android_ver
        self.__ui_ver = ui_ver
        self.__processor = processor

    # getter dan setter untuk android_ver
    def get_android_ver(self):
        return self.__android_ver

    def set_android_ver(self, android_ver):
        self.__android_ver = android_ver

    # getter dan setter untuk ui_ver
    def get_ui_ver(self):
        return self.__ui_ver

    def set_ui_ver(self, ui_ver):
        self.__ui_ver = ui_ver

    # getter dan setter untuk processor
    def get_processor(self):
        return self.__processor

    def set_processor(self, processor):
        self.__processor = processor
